### PR TITLE
libcontainer: configs: extend unsupported os

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca <runcom@redhat.com>

### CODE REVIEW

File: libcontainer/configs/namespaces_unsupported.go [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// +build !linux,!freebsd
```
Comment (runcom) [inline libcontainer/configs/namespaces_unsupported.go]: is `// +build windows,darwin` better?

**Comment by rhatdan** [comment]
@mrunalp PTAL

File: libcontainer/configs/namespaces_unsupported.go (hunk 1 above)
Comment (mrunalp) [inline libcontainer/configs/namespaces_unsupported.go]: I think doing a not is better so we don't have to list every platform. So, we could keep it as is for now.

**Comment by mrunalp** [comment]
LGTM @LK4D4 ping

File: libcontainer/configs/namespaces_unsupported.go (hunk 1 above)
Comment (runcom) [inline libcontainer/configs/namespaces_unsupported.go]: fair enough, I've seen some build tags listing each platform so was a bit concerned

**Comment by LK4D4** [comment]
LGTM

**Pull request merged**